File: 01Conceitos/pessoa.py
# Criação da classe Pessoa
class Pessoa:
    # Construtor
    def __init__(self, nome, idade):
        self.nome = nome
        self.idade = idade

    # Métodos
    def __str__(self):
        return f'{self.nome} tem {self.idade} anos'

    def is_adulto(self):
        return self.idade >= 18

    def aniversario(self):
        self.idade += 1

File: 01Conceitos/Pessoa.cs
using System;

namespace _01Conceitos
{
    class Pessoa
    {
        // Atributos
        public string? Nome { get; set; }
        public int Idade { get; set; }

        // Construtor
        public Pessoa(string nome, int idade)
        {
            Nome = nome;
            Idade = idade;
        }

        // Métodos
        public void Cumprimentar()
        {
            Console.WriteLine($"Olá, meu nome é {Nome} e tenho {Idade} anos.");
        }
    }
}